src/cljs/commiteth/manage_payouts.cljs

```diff
@@ -1,91 +1,356 @@
-    [:div.f2.pa2 (common/usd-string (:combined-usd-value paid))]
-    [:div (:count paid) " bounties"]]])
+   [:div.fl-ns.w-33-ns.tc.pv4
+    [:div.f3.pa2 (:count unpaid)]
+    [:div.ph4 "Open bounties in total"]]])
+
+(defn bounty-stats-new [{:keys [paid unpaid]}]
+  [:div.br3.bg-white.shadow-6.pa4
+   [:span.db.f3.pg-med.mb2 (common/usd-string (:combined-usd-value paid))]
+   [:span.gray "Paid for " [:span.muted-blue (:count paid) " solved bounties"]]
+   [:div.bb.b--near-white.mv3]
+   [:span.db.f3.pg-med.pt1.mb2 (common/usd-string (:combined-usd-value unpaid))]
+   [:span.gray "Open for " [:span.muted-blue (:count unpaid) " bounties"]]])
+
+(def state-mapping
+  {:opened :open
+   :funded :funded
+   :claimed :claimed
+   :multiple-claims :claimed
+   :merged :merged
+   :pending-contributor-address :pending-contributor-address
+   :pending-maintainer-confirmation :pending-maintainer-confirmation
+   :paid :paid})
+
+(defn bounty-title-link [bounty show-date?]
+  [:a {:href (common/issue-url (:repo-owner bounty) (:repo-name bounty) (:issue-number bounty))}
+   [:div.w-100.overflow-hidden
+    [:span.db.f5.pg-med.muted-blue.hover-black (:issue-title bounty)]
+    (when show-date?
+      [:span.db.mt1.f7.gray.pg-book
+       ;;{:on-click #(do (.preventDefault %) (prn (dissoc bounty :claims)))}
+       (common/human-time (:updated bounty))])]])
+
+(defn square-card
+  "A mostly generic component that renders a square with a section
+  pinned to the top and a section pinned to the bottom."
+  [top bottom]
+  [:div.aspect-ratio-l.aspect-ratio--1x1-l
+   [:div.bg-sob-tint.br3.shadow-6.pa3.aspect-ratio--object-l.flex-l.flex-column-l
+    [:div.flex-auto top]
+    [:div bottom]]])
+
+(defn unclaimed-bounty [bounty]
+  [:div.w-third-l.fl-l.pa2
+   [square-card
+    [bounty-title-link bounty true]
+    [:div.f7
+     [bounty-balance bounty]]]])
+
+(defn paid-bounty [bounty]
+  [:div.w-third-l.fl-l.pa2
+   [square-card
+    [:div
+     [bounty-title-link bounty false]
+     [:div.f6.mt1.gray
+      "Paid out to " [:span.pg-med.muted-blue "@" (:winner_login bounty)]]]
+    [:div.f7
+     [bounty-balance bounty]]]])
+
+(defn expandable-bounty-list [bounty-component bounties]
+  (let [expanded? (r/atom false)]
+    (fn expandable-bounty-list-render [bounty-component bounties]
+      [:div
+       [:div.cf.nl2.nr2
+        (for [bounty (cond->> bounties
+                       (not @expanded?) (take 3))]
+          ^{:key (:issue_id bounty)}
+          [bounty-component bounty])]
+       (when (> (count bounties) 3)
+         [:div.tr
+          [:span.f5.sob-blue.pointer
+           {:role "button"
+            :on-click #(reset! expanded? (not @expanded?))}
+           (if @expanded?
+             "Collapse ↑"
+             "See all ↓")]])])))
+
+(defn count-pill [n]
+  [:span.v-mid.ml3.ph3.pv1.bg-light-gray.br3.f7 n])
+
+(defn salute [name]
+  (let [msg-info (rf/subscribe [:dashboard/banner-msg])]
+    (fn salute-render [name]
```

Comment:
this is super helpful to name the render fn, 👍 🎉

Reply:
yeah, :x: to unnamed anon fns ✊